test/e2e/data/e2e_conf.yaml

```
@@ -7,12 +7,6 @@
 managementClusterName: capo-e2e
 
 images:
```

It may be worth it to load the CAPI images here for easier debugging. Also, if running the e2e tests locally, you can save some time if you have the images already on your machine.
The `tryLoad` behavior is that it tries to load the image first from disk, if it cannot be found it is instead pulled.
If that fails, the test is aborted.
```suggestion
images:
- name: gcr.io/k8s-staging-cluster-api/kubeadm-control-plane-controller:nightly_main_20250124
  loadBehavior: tryLoad
...
```